reporters/install.js

```diff
@@ -37,9 +39,13 @@ function summary (data, options) {
     output = output + value + '\n'
   }
 
+  if (!data.advisories) {
+    log(`${yellow('WARN')}: There was an error contacting the audit server`);
```

Comment:
I think build job is failing because of a semicolon on this line.

```
> npm-audit-report@1.3.1 pretest /home/travis/build/npm/npm-audit-report
> standard
standard: Use JavaScript Standard Style (https://standardjs.com)
standard: Run `standard --fix` to automatically fix some problems.
  /home/travis/build/npm/npm-audit-report/reporters/install.js:43:77: Extra semicolon.
npm ERR! Test failed.  See above for more details.
```